src/utils/search/project-search.js

```diff
@@ -1,5 +1,6 @@
 // Maybe reuse file search's functions?
 import { isLoaded } from "../source";
+import React from "react";
```

Comment:
Any reason for importing react here?

Reply:
without that line jest gave the error react not found.
my guess was that the key syntax in the highlight function indirectly requires react.
still kind of weird to me though.